contracts/RiskFund/ReserveHelpers.sol

```diff
@@ -8,6 +8,8 @@ import { SafeERC20Upgradeable } from "@openzeppelin/contracts-upgradeable/token/
 import { ensureNonzeroAddress } from "../lib/validators.sol";
 import { ComptrollerInterface } from "../ComptrollerInterface.sol";
 import { PoolRegistryInterface } from "../Pool/PoolRegistryInterface.sol";
+import { VToken } from "../VToken.sol";
+import { PoolRegistry } from "../Pool/PoolRegistry.sol";
```

Comment:
Instead of importing the whole files. I would recommend to import the interfaces like it's been done in the [`RiskFundConverter`](https://github.com/VenusProtocol/protocol-reserve/blob/e5ad0dffc7ef7b99d57cb9fe8947c9655c6cadc3/contracts/TokenConverter/RiskFundConverter.sol)

Reply:
`PoolRegistryInterface` and `VTokenInterface` are already present. So we can use the same.